src/routes/(app)/admin[[hash=admin_hash]]/config/+page.svelte

```diff
@@ -165,6 +165,16 @@
 		/>
 		Limit product line per cart (for law purpose and small business )
 	</label>
+	<label class="checkbox-label">
+		<input
+			type="checkbox"
+			name="defaultOnLocation"
+			class="form-checkbox"
+			checked={data.defaultOnLocation}
```

Comment:
the "free POS delivery fees" should automatically be checked initially if this checkbox is checked

also when this checkbox changes, the other checkbox should change in consequence